src/storage/mod.rs

```diff
@@ -2401,7 +2402,13 @@ fn prepare_snap_ctx<'a>(
                 .read_key_check(key, |lock| {
                     // No need to check access_locks because they are committed which means they
                     // can't be in memory lock table.
-                    Lock::check_ts_conflict(Cow::Borrowed(lock), key, start_ts, bypass_locks)
+                    Lock::check_ts_conflict(
```

Comment:
Here has a "if" block which is "if isolation_level == IsolationLevel::Si". Should it be modified as "if isolation_level != IsolationLevel::RC"

Reply:
Yes, should be changed to `matches!(isolation_level, IsolationLevel::Si | IsolationLevel::RcCheckTs)`